src/core-layers/experimental/screen-grid-layer/screen-grid-layer-fragment.glsl.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015 - 2017 Uber Technologies, Inc.
```

Comment:
We have an `src/experimental-layers` folder. If you put it there you can even publish it as part of deck.gl layers.